prog/expire_project_invitations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Prog::ExpireProjectInvitations < Prog::Base
+  label def wait
+    ProjectInvitation.all.select { _1.expired_at < Time.now }.each(&:destroy)
```

Comment:
`.all.select` first fetches all records and then selects expired ones. Instead, we can do `ProjectInvitation.where { expired_at < Time.now }`, which would push filtering down to DB.

Reply:
We don't have `expired_at` column at DB. We have only `created_at`. `expired_at` is calculated dynamically. I can write a query like `created_at + 7 days` on the DB side.

Reply:
I decided to keep `expires_at` as column, instead of `created_at` to simplify it

Reply:
Changed it